Examples/Tests/galilean/inputs_3d

```diff
@@ -7,8 +7,7 @@ amr.n_cell = 32 32 32
 amr.max_grid_size = 32
 amr.blocking_factor = 32
 amr.max_level = 0
-psatd.v_galilean = 0. 0. 0.99498743710662
-
+psatd.use_default_v_galilean = 1
```

Comment:
I think this change would have to be reverted, because the input file does not set `warpx.gamma_boost` and the new default option would set zero Galilean velocity, which is not what we want here. Do I understand correctly?
```suggestion
psatd.v_galilean = 0. 0. 0.99498743710662
```

</br>
</br>

However, I think we could use the new parameter for the test `galilean_hybrid_2d`, replacing `runtime_params = psatd.v_galilean = 0. 0. -0.9970370305242862` with `runtime_params = psatd.use_default_v_galilean = 1` in
https://github.com/ECP-WarpX/WarpX/blob/21351adfdbb23ff21a68997fd992153ce437b681/Regression/WarpX-tests.ini#L1597-L1601
I think this should work and could be a good test for the new option.